packages/customize-widgets/src/controls/inspector-section.js

```diff
@@ -57,5 +59,16 @@ export default function getInspectorSection() {
 				allowMultiple: true,
 			} );
 		}
+		collapse( opt ) {
+			this.setIsOpen( false );
+			super.collapse( opt );
+		}
+		setIsOpen( value ) {
```

Comment:
Could use a setter here if you want to be fancy and reduce the change of a future developer accidentally doing `this.isOpen = x` instead of `this.setIsOpen( x )`.

```js
set isOpen( value ) {
	this._isOpen = value;
	this.triggerActiveCallbacks();
}
```